### PR TITLE
[_]:feat/integrate-network-web-package

### PR DESCRIPTION
**Pull request by fabioespinosa**
~**WIP (don't merge):**~

Needs to yarn link the package for @internxt/network-web to work

Issues: 

- Types are not working (from the package), they are not being exported from the package (trying to figure out why)

~Purpose of this PR is just to test the package and approve changes in the package, not in drive-web, not yet~

### CODE REVIEW

File: src/app/network/download/v2.ts [hunk 1]
```diff
@@ -1,34 +1,34 @@
-import { Network } from '@internxt/sdk/dist/network';
 import { sha256 } from '../crypto';
-import { NetworkFacade } from '../NetworkFacade';
+import { NetworkWeb } from '@internxt/network-web/_bundles/prod';
```
Comment (fabioespinosa) [inline src/app/network/download/v2.ts]: not sure if it is a good idea to import it like this (under bundles)

File: src/app/network/upload.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ interface IUploadParams {
   mnemonic: string;
   progressCallback: UploadProgressCallback;
   abortController?: AbortController;
+  parts?: number;
 }
 
 export function uploadFileBlob(
```
Comment (fabioespinosa) [inline src/app/network/upload.ts]: this function should be removed (the package replaces it)

File: src/app/network/download/v2.ts (hunk 1 above)
Comment (fabioespinosa) [reply src/app/network/download/v2.ts]: fixed

**Review by sg-gs — APPROVED** [review]

**Comment by sg-gs** [comment]
We no longer work on this line, fixed with lasts improvements.

**Comment by fabioespinosa** [comment]
Hope it helped a little bit :) :)

**Pull request closed without merging**